integration/util.py

```diff
@@ -945,7 +974,8 @@ def _run_magic_folder(reactor, request, temp_dir, name):
 
     :return Deferred[IProcessTransport]: The started process.
     """
-    config_dir = join(temp_dir, "magic-daemon-{}".format(name))
+    with start_action(action_type=u"integration:run-magic-folder").context():
```

Comment:
I don't understand.  Start a never-finishing action?

Indenting `config_dir` into this block seems like it does nothing except satisfy the syntactic requirement that there be something in the block.  I think `pass` is a slightly less confusing thing to put in the block.  But using `with` at all here seems misleading.  The point of the context manager is to automatically finish the action.  This action never finishes...

If this action has a parent then its parent might finish.  That seems like it would be confusing?  If this action has no parent then will the fact that it never finishes confuse tools that interpret the logs?

Yet ... nothing interesting is ever executed in the context of this action, so all of this is irrelevant.  Can this ever have anything other than a single start message?  If not, why isn't it just a Message?

Reply:
I think this was probably obsolete or a mis-merge from #449.

Reply:
Okay.  It's still there.  Can we fix it?

Reply:
Hmm. I though I had.